tests/phpunit/includes/abstract-testcase.php

```diff
@@ -23,15 +23,14 @@ abstract class WP_UnitTestCase_Base extends PHPUnit_Adapter_TestCase {
 	protected static $hooks_saved = array();
 	protected static $ignore_files;
 
-	public function __isset( $name ) {
-		return 'factory' === $name;
-	}
-
-	public function __get( $name ) {
-		if ( 'factory' === $name ) {
-			return self::factory();
-		}
-	}
+	/**
+	 * Fixture factory.
+	 *
+	 * @deprecated 6.1 Use the WP_UnitTestCase_Base::factory() method instead.
```

Comment:
```suggestion
	 * @deprecated 6.1.0 Use the WP_UnitTestCase_Base::factory() method instead.
```

Reply:
Thanks for catching that. Fixed up in the original commit.